package.json

```diff
@@ -103,10 +103,11 @@
     "precommit": "lint-staged",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "start:storybook": "start-storybook -p 9001 -s ./storybook/static -c storybook/config",
-    "start:webpack": "yarn build -- --watch",
+    "start:webpack": "yarn webpack -- --watch",
```

Comment:
with this change it won't run webpack with production settings. Because we are using different babel aliases and the build time is short no matter what, I think it's worth keeping it running with `-p` even on dev mode